LeadKit.xcodeproj/project.pbxproj

```diff
@@ -527,8 +525,6 @@
 		A6E0DDDE1F8A696F002CA74E /* EmptyCellRow.swift in Sources */ = {isa = PBXBuildFile; fileRef = A66428A71F8A654600C6308D /* EmptyCellRow.swift */; };
 		A6E0DDDF1F8A696F002CA74E /* SeparatorCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = A66428A61F8A653600C6308D /* SeparatorCell.swift */; };
 		A6E0DDE11F8A696F002CA74E /* SeparatorRowBox.swift in Sources */ = {isa = PBXBuildFile; fileRef = A66428A81F8A655600C6308D /* SeparatorRowBox.swift */; };
-		A6E0DDE31F8A696F002CA74E /* SeparatorCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = A66428A61F8A653600C6308D /* SeparatorCell.swift */; };
```

Comment:
Why do you delete this?

Reply:
I removed this file only from "iOS test target" (not project), because of compile errors (unknown types and others)

Reply:
Take a look
![screen shot 2018-05-13 at 20 16 15](https://user-images.githubusercontent.com/17689921/39969825-9e826b16-56ea-11e8-84c2-5315ad60260e.png)

Reply:
Ok. Deal with it